fpdf/html.py

```diff
@@ -738,11 +747,11 @@ def error(self, message):
 class HTMLMixin:
     HTML2FPDF_CLASS = HTML2FPDF
```

Comment:
I think this assignment is redundant now, and can be removed.

Reply:
I deleted it at 62914d0312d32c04d4d320648aa010f4ba0707e3